flink-clients/src/main/java/org/apache/flink/client/cli/CliFrontendParser.java

```diff
@@ -305,6 +307,11 @@ public static void printHelpForSavepoint() {
 		formatter.setSyntaxPrefix("  \"savepoint\" action options:");
 		formatter.printHelp(" ", getSavepointOptions(new Options()));
 		System.out.println();
+		System.out.println("\n  Examples:");
+		System.out.println("  - Trigger savepoint: bin/flink savepoint <Job ID>");
+		System.out.println("  - Dispose savepoint:");
+		System.out.println("    * For a running job: bin/flink savepoint -d <Path> <Job ID>");
+		System.out.println("    * For a terminated job: bin/flink savepoint -d <Path> -j <Jar> [-c <mainClass> -C <classPath>]");
```

Comment:
Is it consistent that we specify the job id without an option whereas we specify the jar with `-j`? Shouldn't only those parameters which are used everywhere be used without an option flag?